notebooks/data_table_transformer/main.py

```diff
@@ -0,0 +1,125 @@
+import os
+import herzog
+
+# Mock the notebook environment
+os.environ['WORKSPACE_NAME'] = "terra-notebook-utils-tests"
+os.environ['WORKSPACE_BUCKET'] = "gs://fc-9169fcd1-92ce-4d60-9d2d-d19fd326ff10"
+os.environ['GOOGLE_PROJECT'] = "firecloud-cgl"
+
+with herzog.Cell("markdown"):
+    """
+    ### Use  Pandas `pivot` command  to convert [AnVIL](https://anvilproject.org) table from  long to wide
+    - `index_column` long format column containing the unique keys of the wide formatted table rows
+    - `header_column` long format column containing the new column headers of the wide formatted table
+    - `value_columns` long format columns to carry over into the wide formatted table (These may be muxed with the
```

Comment:
mux as in multiplex, or is this a typo for "mix"?

Reply:
Will change to “combined”